src/stores/models/PlanModel.js

```diff
@@ -17,8 +16,8 @@ export class PlanModel {
   isPublic = false;
 
   @observable
-  @serializable(list(object(ColorModel)))
-  colorScheme = [];
+  @serializable(map())
+  colorScheme = observable.map({'DEPT': 'green'});
```

Comment:
There's a raw js library in the orig implementation that picks a random nice looking color for you. Just fyi. Could use it here, could not, idrc.

https://github.com/davidmerfield/randomColor

Reply:
We can make it random, but because new courses are automatically given the 'DEPT' department string, they'll all end up being the same color, so randomization really only has value once- and this way we know we don't give the user something too light to read white text against.